pil_analyzer/src/evaluator.rs

```diff
@@ -45,7 +42,6 @@ pub fn compute_constants<T: FieldElement>(
 
 pub struct Evaluator<'a, T> {
```

Comment:
Should we rename this to `ConstantEvaluator`?

Reply:
I would like to move away from the expression "constant". If we want to be more specific, this is an `ExpressionEvaluator` (as opposed to `AlgebraicExpressionEvaluator`), but to me this does not make it much clearer.